src/test/resources/pages/instructorFeedbackResultsPageGQRWithSanitizedData.html

```diff
@@ -462,7 +462,7 @@ <h4 class="modal-title">
                                 <div class="row" id="commentBar-1-1-0-1-1">
                                   <div class="col-xs-10">
                                     <span class="text-muted">
-                                      From: Instructor&lt;script&gt; alert(&#39;hi!&#39;); &lt;&#x2f;script&gt; [Mon, 02 Mar 2026, 07:59 AM SGT] (last edited by Instructor&lt;script&gt; alert('hi!'); &lt;/script&gt; at Tue, 03 Mar 2026, 07:59 AM SGT)
+                                      From: Instructor&lt;script&gt; alert('hi!'); &lt;/script&gt; [Mon, 02 Mar 2026, 07:59 AM SGT] (last edited by Instructor&lt;script&gt; alert('hi!'); &lt;/script&gt; at Tue, 03 Mar 2026, 07:59 AM SGT)
```

Comment:
This change keeps coming even after running this test many times. I don't know why. Shall I revert it?

Reply:
By your changes?

Reply:
No, it's an unstable test. https://github.com/TEAMMATES/teammates/pull/8938/files

Reply:
Then I think you should not introduce it.